docs/manual/advanced-features.tex

```diff
@@ -384,8 +384,8 @@
 locations.
 
 Variables defaulted to top include local variables, resource variables in the
-try-with-resources construct, variables in \<for> statements, and \<catch>
-arguments (known as exception parameters in the Java Language Specification).
+try-with-resources construct, variables in \<for> statements, and exception
+parameters in the \href{https://docs.oracle.com/javase/specs/jls/se17/html/jls-14.html#jls-Catches}{Java Language Specification}.
```

Comment:
The "in the JLS" sounds odd... it's also odd to only have the link for one of them.
Let's remove the JLS reference.

```suggestion
parameters in try blocks.
```

It might be even better to split out "exception parameters" from this list and have a separate item for it, which covers the following two paragraphs.
(Make sure to read those paragraphs!)

Reply:
I think exception parameter should be the parameter in `catch clause`.

Reply:
I have no idea what you mean by this comment.

Reply:
Oh previously, you have code suggestion suggest `exception parameter` in `try blocks`.  I did not commit the suggestion and use `catch clause` instead.

<img width="519" alt="image" src="https://github.com/user-attachments/assets/07dfdde4-1955-4748-b04f-1fdf5a3c47eb">